Documentation/webapp-intro.html

```diff
@@ -92,17 +92,17 @@ <h3>Using the Store</h3>
 <code class="language-javascript">const VCARD = new $rdf.Namespace(‘http://www.w3.org/2006/vcard/ns#‘);
 </code></pre>
 
-<p>If we don’t  know which vocabulary to use, various groups have their favorite lists. One is the <a href="https://github.com/solid/solid-ui/blob/master/src/ns.js">solid-ui list of namespaces</a>.</p>
+<p>If we don’t know which vocabulary to use, various groups have their favorite lists. One is the <a href="https://github.com/solid/solid-ui/blob/master/src/ns.js">solid-ui list of namespaces</a>.</p>
```

Comment:
This link is outdated now (as of https://github.com/solid/solid-ui/commit/95e298d289e5750cfc40c153fa248b93da5f0909#diff-3f43ebf4bdca62f4bf680703836e1e88).

It should link to: https://github.com/solid/solid-namespace/blob/master/index.js

Reply:
Funny how PRs that are ignored for several weeks get a little stale...

Although, as I didn't add that link, it is even staler than that, and fixing it seems like it ought to be its own PR.

Reply:
@TallTed  can you resubmit the PR with the better URL for the namespace mentioned by above : https://github.com/solid/solid-namespace/blob/master/index.js Then I will merge it.

Reply:
@jeff-zucker - I will replace the URL in another PR later today since no-one else seems to want to. (I don't understand why TylerRick, and now you, didn't simply make another PR to fix that stale URL, since it's not related to my PR at all.) 

Please merge this one now, as I cannot change it, as the source repo was inadvertently deleted, as I said earlier today.